api/dataprocessor.go

```diff
@@ -414,11 +414,61 @@ func (s *Server) getSeries(req models.Req, consolidator consolidation.Consolidat
 		// if to < oldest -> no need to search until oldest, only search until to
 		until := util.Min(oldest, toUnix)
 		logLoad("cassan", key, fromUnix, until)
-		storeIters, err := s.BackendStore.Search(key, fromUnix, until)
-		if err != nil {
-			panic(err)
+
+		log.Debug("cache: searching query key %s, from %d, until %d", key, fromUnix, until)
```

Comment:
yeah i haven't documented it yet. i try to have a prefix for each package, more or less.  so you can easily filter logs for the "module" you're interested in (input plugins, index, cache, whatever).
we plan to switch to a different logger soonish, a logger that is actually maintained, and can do structured output, so that's why we don't need to be too strict about the different prefix